app/models/relation.rb

```diff
@@ -0,0 +1,3 @@
+class Relation < ApplicationRecord
```

Comment:
Could this be something more meaningful? e.g. `SchoolRelation` or `RelatedSchool` or even just `SchoolLink` as they are named in GIAS?  `Relation` sounds very low level and `ActiveRecord` like

Reply:
Agree with @tonyheadford here - `Relations` sounds too much like a generic `ActiveRecord` thing. I'd favour `SchoolLink`, but any of the options Tony suggests would be fine

Reply:
Agreed, this is now `SchoolLink`